api/database/factories/UserFactory.php

```diff
@@ -119,7 +119,9 @@ public function definition()
             'accepted_operational_requirements' => $this->faker->optional->randomElements(array_column(OperationalRequirement::cases(), 'name'), 2),
             'gov_employee_type' => $isGovEmployee ? $this->faker->randomElement(GovEmployeeType::cases())->name : null,
             'citizenship' => $this->faker->randomElement(CitizenshipStatus::cases())->name,
-            'armed_forces_status' => $this->faker->randomElement(ArmedForcesStatus::cases())->name,
+            'armed_forces_status' => $this->faker->boolean() ?
+                ArmedForcesStatus::NON_CAF->name
+                : $this->faker->randomElement(ArmedForcesStatus::cases())->name,
```

Comment:
Before 2/3 of users were current/past armed forces which seemed a bit egregiously high. This small tweak just halves it.